src/System.Text.Json/tests/Serialization/EnumTests.cs

```diff
@@ -394,5 +394,146 @@ public static void EnumAsSByteMax()
             SimpleTestClass obj = JsonSerializer.Parse<SimpleTestClass>(s_jsonSByteEnum);
             Assert.Equal(SampleSByteEnum.Max, obj.MySByteEnum);
         }
+
+        [Fact]
+        public static void ToString_ZeroEnum_ReturnsZero()
+        {
+            string json = JsonSerializer.ToString((SampleByteEnum)0);
+            Assert.Equal("0", json);
+            json = JsonSerializer.ToString((SampleSByteEnum)0);
+            Assert.Equal("0", json);
+            json = JsonSerializer.ToString((SampleEnumUInt16)0);
+            Assert.Equal("0", json);
+            json = JsonSerializer.ToString((SampleUInt32Enum)0);
+            Assert.Equal("0", json);
+            json = JsonSerializer.ToString((SampleUInt64Enum)0);
+            Assert.Equal("0", json);
+            json = JsonSerializer.ToString((SampleInt64Enum)0);
+            Assert.Equal("0", json);
+        }
+
+        [Fact]
+        public static void ToString_MaxValue_ReturnsMaxValue()
+        {
+            string json = JsonSerializer.ToString((SampleByteEnum)byte.MaxValue);
+            Assert.Equal(byte.MaxValue.ToString(), json);
+            json = JsonSerializer.ToString((SampleSByteEnum)sbyte.MaxValue);
+            Assert.Equal(sbyte.MaxValue.ToString(), json);
+            json = JsonSerializer.ToString((SampleEnumUInt16)ushort.MaxValue);
+            Assert.Equal(ushort.MaxValue.ToString(), json);
+            json = JsonSerializer.ToString((SampleUInt32Enum)UInt32.MaxValue);
+            Assert.Equal(UInt32.MaxValue.ToString(), json);
+            json = JsonSerializer.ToString((SampleUInt64Enum)UInt64.MaxValue);
+            Assert.Equal(UInt64.MaxValue.ToString(), json);
+            json = JsonSerializer.ToString((SampleInt64Enum)long.MaxValue);
+            Assert.Equal(long.MaxValue.ToString(), json);
+        }
+
+        [Fact]
+        public static void ToString_MinValue_ReturnsMinValue()
```

Comment:
nit: Since we have specific tests for each numeric type below (like `ToString_Byte`), why not add 0, minValue, maxValue to those instead as `InlineData` and remove these test cases (`ToString_ZeroEnum_ReturnsZero`, `ToString_MaxValue_ReturnsMaxValue`, `ToString_MinValue_ReturnsMinValue`)?

Reply:
Removed the duplicate tests. In a separate PR I can further cleanup the Parse tests as well, that can remove duplicate and add missing test cases and cleanup the overall file